spec/dummy/app/api/api.rb

```diff
@@ -26,5 +26,10 @@ class API < Grape::API
     request.params.as_json
   end
 
+  desc 'Get Authorization header.', authorizations: { oauth2: [] }
```

Comment:
I can set it up but it'll add the dep of doorkeeper (and activerecord). Okay, sounds good.